README.md

```diff
@@ -264,6 +264,8 @@ For the `FS` persistor, the `bucketName` should be the full path to the folder o
 - `s3.key` (required): The AWS access key ID
 - `s3.secret` (required): The AWS secret access key
 - `s3.partSize`: The part size for S3 uploads. Defaults to 100 megabytes.
+- `s3.httpOptions`: HTTP options passed directly to the [S3 constructor](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#constructor-property).
```

Comment:
What options are you envisaging here? It feels like it might be something one might want to pass to the methods themselves, although that would mean polluting the generic method options with platform-specific things so I am in two minds.

Reply:
v1 history sets a global timeout: https://github.com/overleaf/write_latex/blob/master/history/storage/lib/s3.js#L11